api/locale/ba.yml

```diff
@@ -0,0 +1,20 @@
+ba:
```

Comment:
If this is Bengali, I believe the 2-letter code is `bn` (see https://gist.github.com/DenverCoder1/f61147ba26bfcf7c3bf605af7d3382d5)